apolline-flutter/lib/sensor_view.dart

```diff
@@ -168,20 +191,41 @@ class _SensorViewState extends State<SensorView> {
       List<int> userThresholds = widget.ucS.userConf.getThresholds(value);
       int warningThreshold = userThresholds[0];
       int dangerThreshold = userThresholds[1];
+      if(widget.ucS.userConf.showWarningNotifications && SensorSendNegativeValue().isSatisfiedBy(collectedValue)) { // send notification if negative value is detected
```

Comment:
How can I test this?

Reply:
I didn't have negative value with my sensor but apparently, it's possible to have it when you put your sensor above a tea cup or coffee for example.
_If you to be sure that class works you can see tests_

Reply:
Yeah I saw your tests; I'd like to be able to display inconsistency report form from a test.